adapters/consumable/consumable.go

```diff
@@ -89,12 +89,14 @@ func (a *adapter) MakeBids(request *openrtb2.BidRequest, requestData *adapters.R
 
 	bidResponse := adapters.NewBidderResponseWithBidsCapacity(len(request.Imp))
 	bidResponse.Currency = response.Cur
-	var errors []error
+	// var errors []error // TODO(dmitris) - see below, find out why we need to collect these errors
 	for _, seatBid := range response.SeatBid {
 		for i, bid := range seatBid.Bid {
 			bidType, err := getMediaTypeForBid(bid)
 			if err != nil {
-				errors = append(errors, err)
+				// golangci-lint: SA4010: this result of append is never used, except maybe in other appends (staticcheck)
+				// TODO(dmitris) - confirm if we need to collect these errors and if yes, find out why
+				// errors = append(errors, err)
```

Comment:
Can you explain these TODOs? Also, the commented out lines on 92 and 99, i think should be addressed. Either deleted or uncommented.

Reply:
I hoped that maybe some of the developers more familiar with the codebase and its background/history would chime in, but otherwise I deleted the `TODO`s and the commented out lines in [befeec1](https://github.com/prebid/prebid-server/commit/befeec1073df522d998b2f1c1e118d4931671b3a)

Reply:
@bsardo Please contact the Consumable folks about these lint violations. It's most appropriate for them to address.